src/const.h

```diff
@@ -85,3 +85,5 @@
 
 #define DEFAULT_XML_LOG_FILE_NAME	"ntttcp-for-linux-log.xml"
 #define DEFAULT_CONSOLE_LOG_FILE_NAME	"ntttcp-for-linux-log.log"
+#define DEFAULT_JSON_LOG_FILE_NAME	"ntttcp-for-linux-log.json"
```

Comment:
Instead of adding this line after the line for "ntttcp-for-linux-log.log", can we arrange those 3 log files in this order: "ntttcp-for-linux-log.log", then "ntttcp-for-linux-log.xml", then "ntttcp-for-linux-log.json". The same order for below changes too ("-o", "-x", "-j").

Reply:
Has been modified